doc/ref/lists.xml

```diff
@@ -1815,8 +1815,8 @@ An <E>enumerator</E> is an immutable list that need not store its elements
 explicitly but knows, from a set of basic data,
 how to determine the <M>i</M>-th element and the position of a given object.
 A typical example of this is a vector space over a finite field with <M>q</M>
-elements, say, for which it is very easy to enumerate all elements
-using <M>q</M>-adic expansions of integers.
+elements, where <M>q</M> is a prime power, for which it is very easy to
+enumerate all elements using <M>q</M>-adic expansions of integers.
```

Comment:
This is one of those points where I show my mathematical ignorance:

I would have guessed that q-adic expansions are the same as p-adic expansions, and I would have guessed that the p of p-adic is required to be a prime.

Is that the case? Basically: does it make sense to talk about a (prime power)-adic expansion?

If not, this sentence should be adjusted to say that q must be a prime/the finite field must be prime.

Reply:
You're right, I'm sorry, I didn't pay enough attention there. Will fix it.